inlang/source-code/sdk/src/loadProject.test.ts

```diff
@@ -1145,11 +1162,25 @@ describe("functionality", () => {
 
 			messages.data[0]!.variants[0]!.pattern[0]!.value = "changed3"
 
-			// change file
+			// change file - update message
 			await fs.writeFile("./messages.json", JSON.stringify(messages))
 			await new Promise((resolve) => setTimeout(resolve, 200)) // file event will lock a file and be handled sequentially - give it time to pickup the change
 
 			expect(counter).toBe(3)
+
+			// change file - add a message
+			messages.data.push(newMessage)
+			await fs.writeFile("./messages.json", JSON.stringify(messages))
+			await new Promise((resolve) => setTimeout(resolve, 200)) // file event will lock a file and be handled sequentially - give it time to pickup the change
+
+			expect(counter).toBe(4)
```

Comment:
In this case, instead of just checking the event counter, I think the test should check the number of messages returned from `inproject.query.messages.getAll.subscribe(()`.

Reply:
test added

Reply:
much better -  thank you.